CenterForOpenScience/osf.io
# [ENG-1117] No logs for children

## Purpose

Logs for children are worse than good: they're bad. Getting the first page logs for RPP takes about 16 seconds at the best of times, and if that request were made multiple times, or if it happened during a time of heavy load, it could cause a lot of problems for the system.

## Changes

Remove roll-up of child logs into the aggregated log query and instead just return the node being requested (minus the hidden logs).

## QA Notes
Prep

For prep, you'll want to have a complex node hierarchy, and you'll want to see how long that log list takes to load. This can be tested in the front-end via the project summary page and the api (/v2/nodes/<node_id>/logs).
How the end-user will see this

The major changes will be that the node should load faster, and that only the logs for the current node should show in the log widget. Everything else should remain the same.
Potential other things

On the off chance of something weird, do a registration of a multi-node project hierarchy and make sure that works properly.

## Documentation

N/A

## Side Effects

Could maybe affect how registrations register, but I can't see evidence in the code to support that.
Ticket

Research ticket: https://openscience.atlassian.net/browse/ENG-1117
